vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManagerTest.java

```diff
@@ -0,0 +1,74 @@
+package edu.asu.diging.vspace.core.services.impl;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.dao.EmptyResultDataAccessException;
+
+import edu.asu.diging.vspace.core.data.ImageContentBlockRepository;
+import edu.asu.diging.vspace.core.data.TextContentBlockRepository;
+import edu.asu.diging.vspace.core.exception.BlockDoesNotExistException;
+
+public class ContentBlockManagerTest {
+    @Mock
+    private TextContentBlockRepository textBlockRepo;
+
+    @Mock
+    private ImageContentBlockRepository imageBlockRepo;
+
+    @InjectMocks
+    private ContentBlockManager managerToTest;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+
+    }
+
+    @Test
+    public void test_deleteTextBlockById_success() throws BlockDoesNotExistException {
+        String textBlockId = "2";
+        managerToTest.deleteTextBlockById(textBlockId);
+        Mockito.verify(textBlockRepo).deleteById(textBlockId);
+    }
+
+    @Test(expected = BlockDoesNotExistException.class)
+    public void test_deleteTextBlockById_forNonExistentId() throws BlockDoesNotExistException {
+        String textBlockId = "notARealId";
+        Mockito.doThrow(EmptyResultDataAccessException.class).when(textBlockRepo).deleteById(textBlockId);
+        managerToTest.deleteTextBlockById(textBlockId);
+    }
+
+    @Test
+    public void test_deleteTextBlockById_whenIdIsNull() throws BlockDoesNotExistException {
+        String textBlockId = "2";
+        managerToTest.deleteTextBlockById(null);
+        Mockito.verify(textBlockRepo, Mockito.never()).deleteById(textBlockId);
```

Comment:
textblockid is '2' here, so this would indeed never be called (since you're calling it with null). 

That's why one shouldn't copy and paste ;)